es-app/src/SystemScreenSaver.cpp

```diff
@@ -269,6 +269,13 @@ void SystemScreenSaver::renderScreenSaver()
 			Transform4x4f transform = Transform4x4f::Identity();
 			mVideoScreensaver->render(transform);
 		}
+
+		// Check if slideshow then loop background music
+		if (screensaver_behavior == "slideshow" && mBackgroundAudio && !mBackgroundAudio->isPlaying())
+		{
+			mBackgroundAudio->play();
```

Comment:
Will this loop background music while playing the video? And if so, is that the intended behavior?

Reply:
It can. If a background music file is provided. If background music is wanted also during video snaps, then the video audio should be muted (in the Sound Settings menu or Video Screensaver menu) otherwise the audio tracks will overlap.

The menus are a bit convoluted tbh, as also another user in the forum has noted. I am leaning toward to restructure the menus

Reply:
Ok.